e2e/e2etests/test_solana_deposit.go

```diff
@@ -23,7 +29,7 @@ func TestSolanaDeposit(r *runner.E2ERunner, args []string) {
 	require.NoError(r, err)
 
 	// create 'deposit' instruction
-	instruction := r.CreateDepositInstruction(privkey.PublicKey(), r.EVMAddress(), depositAmount.Uint64())
+	instruction := r.CreateDepositInstruction(privkey.PublicKey(), r.EVMAddress(), nil, depositAmount.Uint64())
```

Comment:
Not a block for this but in the future, we should offset most logic in sub utility functions and keep the actual file minimal like https://github.com/zeta-chain/node/blob/c5868affc70b75fbb02ff80de4dc1150abd504de/e2e/e2etests/test_v2_erc20_withdraw_and_call.go#L28

Reply:
refactor in commit https://github.com/zeta-chain/node/pull/2726/commits/92d67f5897576fc119b93838893b0aaf9a43ec08